fe/fe-core/src/main/java/com/starrocks/lake/compaction/PartitionStatistics.java

```diff
@@ -32,6 +32,25 @@ public class PartitionStatistics {
     @SerializedName(value = "compactionScore")
     private Quantiles compactionScore;
 
+    // default priority is -1, manual compaction will have priority value 1
+    @SerializedName(value = "priority")
+    private volatile CompactionPriority priority = CompactionPriority.DEFAULT;
```

Comment:
Will this initialization assignment take effect when deserializing from previously serialized data without this field?

Reply:
Yes, that should be the expected behavior. I will add a ut for this case.

Reply:
> Will this initialization assignment take effect when deserializing from previously serialized data without this field?

Gson will not ensure backward compatibility here. Added some check logic in `getPriority()` method to ensure it.